<launch>
    <node name="driver" pkg="jaguar" type="diff_drive">
        <!-- CAN bus Configuration -->
        <param name="port" value="/dev/jaguar"/>
        <param name="id_left"  value="2"/>
        <param name="id_right" value="3"/>
        <!-- Periodic Message Rates -->
        <param name="heartbeat" value="50"/> <!-- period in ms -->
        <param name="status"    value="50"/> <!-- period in ms -->
        <!-- Robot Model Parameters -->
        <param name="cpr" value="230"/>
        <param name="wheel_diameter" value="0.254"/> <!-- meters -->
        <param name="wheel_separation" value="0.838"/> <!-- meters -->
        <!-- PID Gains -->
        <param name="gain_p" value="1.00"/>
        <param name="gain_i" value="0.02"/>
        <param name="gain_d" value="0.00"/>
        <!-- Odometry TF Frames -->
        <param name="frame_parent" value="/map"/>
        <param name="frame_child"  value="/odom"/>
        <remap from="/cmd_vel" to="/drive/cmd_vel"/>
        <!-- Acceleration Limits -->
        <param name="accel_max" value="10"/>
    </node>
</launch>
